src/3D/LandIsland.cpp

```diff
@@ -58,7 +62,7 @@ void LandIsland::LoadFromFile(const std::filesystem::path& path)
 	_landBlocks.resize(lndBlocks.size());
 	for (size_t i = 0; i < _landBlocks.size(); i++)
 	{
-		_landBlocks[i]._block = std::make_unique<lnd::LNDBlock>(lndBlocks[i]);
+		_landBlocks[i].GetLndBlock() = std::make_unique<lnd::LNDBlock>(lndBlocks[i]);
```

Comment:
Not a big fan of this. Should be using a SetLndBlock function.

Reply:
Fixed on next commit